happi/errors.py

```diff
@@ -21,3 +21,16 @@ class ContainerError(Exception):
 class SearchError(Exception):
     """Raised when no device is found while searching."""
     pass
+
+
+class EnforceError(ValueError):
+    """Raised when value fails a value fails enforcement checks."""
```

Comment:
```suggestion
    """Raised when a value fails enforcement checks."""
```

Reply:
sigh sometimes I think my eyeballs are just for show

Reply:
I do this literally every PR